tests/test_formula.py

```diff
@@ -98,3 +215,161 @@ def test_formulas(
         with pytest.raises(type(exception), match=re.escape(str(exception))):
             formula = func_wrapper()
             formula.validate()
+
+
+formula_snql_tests = [
```

Comment:
Up to you, but I think it'd help me better understand which formula queries aren't supported if we add invalid tests (e.g. different groupbys among timeseries, etc.)

Reply:
I added those to the formula tests above. These tests are now simply testing the serialization. The regular formula tests are testing all the cases I added.